### YamlMime:ManagedReference
items:
- uid: System.Runtime.Remoting.RemotingException
  id: RemotingException
  children:
  - System.Runtime.Remoting.RemotingException.#ctor
  - System.Runtime.Remoting.RemotingException.#ctor(System.String)
  - System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RemotingException
  nameWithType: RemotingException
  fullName: System.Runtime.Remoting.RemotingException
  type: Class
  summary: "时出现了问题在远程处理期间，将引发异常。"
  remarks: "RemotingException 使用 HRESULT COR_E_REMOTING，其值 0x8013150B。       RemotingException 使用默认<xref:System.Object.Equals%2A>支持引用相等性的实现。</xref:System.Object.Equals%2A>       RemotingException 的实例的初始属性值的列表，请参阅 RemotingException 构造函数。"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class RemotingException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor
  id: '#ctor'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException()
  nameWithType: RemotingException.RemotingException()
  fullName: System.Runtime.Remoting.RemotingException.RemotingException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新实例<xref href=&quot;System.Runtime.Remoting.RemotingException&quot;></xref>使用默认属性的类。"
  remarks: "<xref:System.Runtime.Remoting.RemotingException>继承<xref:System.SystemException>类。</xref:System.SystemException></xref:System.Runtime.Remoting.RemotingException> 下表显示<xref:System.Runtime.Remoting.RemotingException>类的实例的初始属性值: |属性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`|  |<xref:System.Exception.Message%2A>|空字符串 (&quot;&quot;) |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException ();
    parameters: []
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(String)
  nameWithType: RemotingException.RemotingException(String)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新实例<xref href=&quot;System.Runtime.Remoting.RemotingException&quot;></xref>使用指定的消息的类。"
  remarks: "<xref:System.Runtime.Remoting.RemotingException>继承<xref:System.SystemException>类。</xref:System.SystemException></xref:System.Runtime.Remoting.RemotingException> 下表显示<xref:System.Runtime.Remoting.RemotingException>类的实例的初始属性值: |属性类型 |条件 |  |-------------------|---------------|  |<xref:System.Exception.InnerException%2A>|`null`|  |<xref:System.Exception.Message%2A>|`message`字符串 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException (string message);
    parameters:
    - id: message
      type: System.String
      description: "解释异常发生原因的错误消息。"
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(SerializationInfo,StreamingContext)
  nameWithType: RemotingException.RemotingException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新实例<xref href=&quot;System.Runtime.Remoting.RemotingException&quot;></xref>从序列化数据的类。"
  remarks: "在反序列化，以便重建通过流传输的异常对象过程中调用此构造函数。 (在序列化的详细信息，请参阅[XML 和 SOAP 序列化](~/add/includes/ajax-current-ext-md.md)。)"
  syntax:
    content: protected RemotingException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "承载序列化的对象数据的对象。"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "有关源或目标的异常的上下文信息。"
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "The <code>info</code> parameter is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.Remoting.RemotingException
  langs:
  - csharp
  name: RemotingException(String,Exception)
  nameWithType: RemotingException.RemotingException(String,Exception)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新实例<xref href=&quot;System.Runtime.Remoting.RemotingException&quot;></xref>使用指定的错误消息和对导致此异常的内部异常的引用的类。"
  remarks: "前一个异常的直接结果将包含中的前一个异常的引用引发的异常，<xref:System.Exception.InnerException%2A>属性。</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>属性返回相同的值传递到构造函数中，或`null`如果<xref:System.Exception.InnerException%2A>属性不会提供给构造函数的内部异常值。</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       下表显示<xref:System.Runtime.Remoting.RemotingException>类的实例的初始属性值: |属性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|内部异常引用。 |  |<xref:System.Exception.Message%2A>|错误消息字符串。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.RemotingException>"
  syntax:
    content: public RemotingException (string message, Exception InnerException);
    parameters:
    - id: message
      type: System.String
      description: "解释异常发生原因的错误消息。"
    - id: InnerException
      type: System.Exception
      description: "导致当前异常的异常。 如果`InnerException`参数不是<xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>中, 引发当前异常<xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;></xref>处理内部异常的块。"
  overload: System.Runtime.Remoting.RemotingException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.Runtime.Remoting.RemotingException.#ctor
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException()
  nameWithType: RemotingException.RemotingException()
  fullName: System.Runtime.Remoting.RemotingException.RemotingException()
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(String)
  nameWithType: RemotingException.RemotingException(String)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(SerializationInfo,StreamingContext)
  nameWithType: RemotingException.RemotingException(SerializationInfo,StreamingContext)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Runtime.Remoting.RemotingException.#ctor(System.String,System.Exception)
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException(String,Exception)
  nameWithType: RemotingException.RemotingException(String,Exception)
  fullName: System.Runtime.Remoting.RemotingException.RemotingException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Runtime.Remoting.RemotingException.#ctor*
  parent: System.Runtime.Remoting.RemotingException
  isExternal: false
  name: RemotingException
  nameWithType: RemotingException.RemotingException
